dispatcher/stateful.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function StatefulDispatcher(dispatcher, data, onData) {
```

Comment:
+1 I also wonder when I should use a `StaticDispatcher` and when a `StatefulDispatcher`.